src/apps/applets/service/user_applet_access.py

```diff
@@ -0,0 +1,140 @@
+        """
+        access = await UserAppletAccessCRUD().get_by_roles(
+            self._user_id, self._applet_id, [Role.ADMIN, Role.MANAGER]
+        )
+        return getattr(access, "role", None)
+
+    async def is_respondents_manager(self) -> Role | None:
+        """
+        Checks whether user is in respondents manager group and returns role
+
+        Permissions:
+        - Invite new respondent
+        - View all respondents
+        - Remove specific respondent access
+        - Invite new reviewer to specific respondent
+        - Set schedule/notifications for respondents
+        """
+        access = await UserAppletAccessCRUD().get_by_roles(
+            self._user_id,
+            self._applet_id,
+            [Role.ADMIN, Role.MANAGER, Role.COORDINATOR],
+        )
+        return getattr(access, "role", None)
+
+    async def is_editor(self) -> Role | None:
+        """
+        Checks whether user is in editor group and returns role
+
+        Permissions:
+        - Create applets
+        - Update applets
+        - Can view all applets
+        # TODO: which applets, assigned or all applets in organization
+        """
+        access = await UserAppletAccessCRUD().get_by_roles(
+            self._user_id,
+            self._applet_id,
+            [Role.ADMIN, Role.MANAGER, Role.EDITOR],
+        )
+        return getattr(access, "role", None)
+
+    async def is_reviewer(self):
+        """
+        Checks whether user is in reviewer group and returns role
+
+        Permissions:
+        - View/Export all respondents' data
+        - Delete specific respondents' data
+        """
+        access = await UserAppletAccessCRUD().get_by_roles(
+            self._user_id, self._applet_id, [Role.ADMIN, Role.MANAGER]
+        )
+        return getattr(access, "role", None)
+
+    async def is_reviewer_for_respondent(self):
+        """
+        Checks whether user is in reviewer for
+          respondent group and returns role
+
+        Permissions:
+        - View assigned respondents' data
+        - Export assigned respondents' data
+        """
+        access = await UserAppletAccessCRUD().get_by_roles(
+            self._user_id,
+            self._applet_id,
+            [Role.ADMIN, Role.MANAGER, Role.REVIEWER],
+        )
+        return getattr(access, "role", None)
+
+    async def is_respondent(self):
+        """
+        Checks whether user is in respondent group and returns role
+
+        Permissions:
+        - Answer to applet
+        """
+        access = await UserAppletAccessCRUD().get_by_roles(
+            self._user_id,
+            self._applet_id,
+            [
+                Role.ADMIN,
+                Role.MANAGER,
+                Role.COORDINATOR,
+                Role.EDITOR,
+                Role.REVIEWER,
+                Role.RESPONDENT,
+            ],
+        )
+        return getattr(access, "role", None)
```

Comment:
Don't you think that `is_admin` and the rest of the methods sound more like a property and it should return the boolean value?